sources/lib/build-system/build.dylan

```diff
@@ -69,22 +69,24 @@ end macro;
 
 
 
-// Toplevel internal function that can be invoked by Dylan Clients
-
+// Top level internal function that can be invoked by Dylan Clients
+// Arguments:
+// build-targets - A sequence of strings naming libraries to build.
+// directory - The directory in which to perform the build.
+// progress-callback - A function to call with build progress messages.
+//     Called with the message string and `warning?: #t` if the message
+//     is a warning.
+// build-script - Jam build script <file-locator>
+// compiler-back-end - #"llvm", #"c", #"harp", or #f.  #f means use HARP.
```

Comment:
``#f`` doesn't mean use HARP, it just means the information is not provided or irrelevant. The actual back-end choice is made in ``make-project`` in the ``projects`` library.

Reply:
Under what conditions is the back-end irrelevant?